src/codes.ts

```diff
@@ -37,9 +40,9 @@ const codeList: ReadonlyArray<Readonly<ICode>> = Object.freeze((
   ] as Array<[ChainCode, UnspentType, Purpose]>
 ).map(([id, type, purpose]) => Object.freeze({ id, type, purpose })));
 
-export const ChainType = tcomb.irreducible('ChainType', (n) => isValid(n));
```

Comment:
I'm not so certain this works as expected

`export default { foo }` means you can do `import Codes from 'unspents/codes'` and use `Codes.foo`.

`export const foo` means you can do `import { foo } from 'unspents/codes'`.

Please double-check that the latter still works.

Reply:
You're right. Reverted.